### ISSUE TITLE
unused variable

### ISSUE THREAD

**Issue by geraldodev** [multunus/dashboard-clj#3]
Hi,

Thank you for dashboard-clj.
It's look like https://github.com/multunus/dashboard-clj/blob/c6a41861d54c63f48448b507ff06bf012b9b835e/src/clj/dashboard_clj/components/webserver.clj#L8 is not used.

Regards,
geraldodev

**Comment by tacticiankerala** [comment]
Nice catch @geraldodev 👍. 

We'll fix this over the weekend. Thanks for reporting :)

**Comment by geraldodev** [comment]
Hi,

Thank you!

In fact I've unpacked the code of dashboard-clj to alter its system-map.
I have a database dependency and want to pass it as a context map on the
1st parameter of the fetcher functions.
So I've coded a Context component, which receives the database connection
and the raw edn of data-sources.

Context component
https://gist.github.com/geraldodev/501440b313ed3c0de80d073d923877df

DataSource
https://gist.github.com/geraldodev/3f2bdc1509bc293f01c71529dec7edc3

system-map
https://gist.github.com/geraldodev/ec020350f8b7c3efed2b0bf90b54ae22

2016-08-11 9:24 GMT-03:00 Sreenath Nannat notifications@github.com:
